parquet-hadoop/src/test/java/org/apache/parquet/filter2/statisticslevel/TestStatisticsFilter.java

```diff
@@ -297,6 +329,26 @@ public boolean keep(Integer value) {
     }
   }
 
+  public static class AllPositiveUdp extends UserDefinedPredicate<Double> {
+    @Override
+    public boolean keep(Double value) {
+      if (value == null) {
```

Comment:
Should/could this be an assert instead?

Reply:
It is more an UDP implementation for testing purposes than the test itself. In addition, we have to return a boolean even if an assertion is added.